go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/IBM/go-sdk-core
+
+go 1.12
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
Was the `incompatible` verison autogenerated?

Reply:
yes, the `go mod tidy` command must have found that one as the most recent version.
I'll check to see if there's a better version that we should be using instead.

Reply:
ok, I initially thought that the `+incompatible` part of that version tag was to be feared.  But it turns out that the go mod command adds that part automatically in certain situations.   I don't fully understand it, but I don't think it's a problem.   The most recent version of that package (as seen on github) is "v3.2.0", so we should be good.